content/administration/upgrade/odoo_online.rst

```diff
@@ -11,15 +11,18 @@ Requesting a test upgrade
    .. image:: odoo_online/accessing-my-databases.png
       :alt: Selecting My Databases under my profile
 
-#. Click on the action settings icon next to your main database and select the *Upgrade* option.
+#. Click on the :guilabel:`Up arrow button` next to your main database name to proceed to the test
```

Comment:
I think you can leave it out of the guilabel, since it's a description of the button